Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -72,7 +72,7 @@ Worker_RequestId USpatialSender::CreateEntity(const FString& ClientWorkerId, con
 
 		ComponentWriteAcl.Add(ClassInfo->SingleClientComponent, WorkersOnly);
 		ComponentWriteAcl.Add(ClassInfo->MultiClientComponent, WorkersOnly);
-		// Not adding handover since component's handover properties will be handled by the actor anyway
+		ComponentWriteAcl.Add(ClassInfo->HandoverComponent, WorkersOnly);
```

Comment:
I'm thinking since we do `RPCComponents[RPC Type]`, maybe we could do the same with data components?

Reply:
Same as with the enum for data components, I think it could be nice, but not a priority.